#!/usr/bin/env python
# set up dynamically reconfigurable parameters here 
PACKAGE = "waypoint_updater"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# 759
gen.add("dyn_test_stoplight_wp",   int_t, 0, "TESTING next stopline wp",
        470, 0, 10902)
gen.add("dyn_update_rate",         int_t, 0, "An Integer parameter",
        10, 1, 50)
gen.add("dyn_lookahead_wps",       int_t, 0, "number of final waypoints",
        50, 10,  200)
gen.add("dyn_default_velocity", double_t, 0, "speed to use",
        20.0, 0.0, 100.0)
gen.add("dyn_default_accel",    double_t, 0, "maximum accel deccel to use",
        1.2, 0.0, 10.0)

exit(gen.generate(PACKAGE, "waypoint_updater", "DynReconf"))
